### PR TITLE
add notebook for colab

### PR DESCRIPTION
**Pull request by rickycao-qy**
add notebooks for google colab

### CODE REVIEW

File: notebooks/pipcook_image_classification.ipynb [hunk 1]
```diff
@@ -0,0 +1,332 @@
...
+        "\n",
+        "Note that the prediction result we give is the probability of each category. You can process this probability to the result you want.\n",
+        "\n",
+        "## Conclusion\n",
+        "\n",
+        "In this way, the component recognition task based on the image classification model is completed. After completing the pipeline in our example, if you are interested in such tasks, you can also start preparing your own dataset for training. We have already introduced the format of the dataset in detail in the data preparation chapter. You only need to follow the file directory to easily prepare the data that matches our image classification pipeline."
+      ]
+    }
+  ]
+}
```
Comment (yorkie) [inline notebooks/pipcook_image_classification.ipynb:332]: EOF

File: notebooks/pipcook_object_detection.ipynb [hunk 2]
```diff
@@ -0,0 +1,410 @@
...
+        "\n",
+        "After making the above dataset, organize the file structure according to the introduction in the previous chapter. After that, you can start the pipeline for training.\n",
+        "\n",
+        "## Conclusion\n",
+        "\n",
+        "Readers have learned how to identify multiple front-end components in a image, which can be applied to some more general scenarios. So in an article, we will introduce a more interesting example, how to use Pipcook to achieve the transfer of picture style, such as replacing the oranges in the picture with apples, or replacing the realistic photo style with oil painting style."
+      ]
+    }
+  ]
+}
```
Comment (yorkie) [inline notebooks/pipcook_object_detection.ipynb:410]: Ditto

File: docs/zh-cn/tutorials/component-object-detection.md [hunk 3]
```diff
@@ -1,5 +1,7 @@
 # 识别图片中的前端组件
 
+使用 **Google Colab Notebook** 的免费GPU训练: <a href="https://colab.research.google.com/github/alibaba/pipcook/blob/master/notebooks/pipcook_object_detection.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a>
```
Comment (yorkie) [inline docs/zh-cn/tutorials/component-object-detection.md:3]:
```suggestion
使用 **Google Colab Notebook** 的免费 GPU/TPU 训练: <a href="https://colab.research.google.com/github/alibaba/pipcook/blob/master/notebooks/pipcook_object_detection.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a>
```

File: docs/zh-cn/tutorials/component-image-classification.md [hunk 4]
```diff
@@ -1,5 +1,7 @@
 # 分类图片中的前端组件
 
+使用 **Google Colab Notebook** 的免费GPU训练: <a href="https://colab.research.google.com/github/alibaba/pipcook/blob/master/notebooks/pipcook_image_classification.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a>
```
Comment (yorkie) [inline docs/zh-cn/tutorials/component-image-classification.md:3]:
```suggestion
使用 **Google Colab Notebook** 的免费 GPU/TPU 训练: <a href="https://colab.research.google.com/github/alibaba/pipcook/blob/master/notebooks/pipcook_image_classification.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a>
```

File: notebooks/pipcook_image_classification.ipynb (hunk 1 above)
Comment (rickycao-qy) [reply notebooks/pipcook_image_classification.ipynb]: fixed

File: notebooks/pipcook_object_detection.ipynb (hunk 2 above)
Comment (rickycao-qy) [reply notebooks/pipcook_object_detection.ipynb]: fixed

File: docs/zh-cn/tutorials/component-object-detection.md (hunk 3 above)
Comment (rickycao-qy) [reply docs/zh-cn/tutorials/component-object-detection.md]: done

File: docs/zh-cn/tutorials/component-image-classification.md (hunk 4 above)
Comment (rickycao-qy) [reply docs/zh-cn/tutorials/component-image-classification.md]: done

**Pull request merged**